### PR TITLE
Adding periodic scan

### PR DESCRIPTION
**Pull request by sshniro**

### CODE REVIEW

File: .github/workflows/zap-scan.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: master
+      - name: ZAP Scan
+        uses: zaproxy/action-baseline@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          target: 'https://www.example.com/'
```
Comment (psiinon) [inline .github/workflows/zap-scan.yml:18]: shouldnt the target be https://www.zaproxy.org/ ??

Comment (sshniro) [reply .github/workflows/zap-scan.yml]: aw~ wrong branch, anyway updated this branch as well.

File: .github/workflows/zap-scan.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+on:
+  schedule:
+    - cron:  '*/10 * * * *'
+
+jobs:
+  zap_scan:
+    runs-on: ubuntu-latest
+    name: Scan the webapplication
+    steps:
+      - name: Checkout
```
Comment (thc202) [inline .github/workflows/zap-scan.yml:10]: We could remove this step, not really needed.

Comment (sshniro) [reply .github/workflows/zap-scan.yml:10]: We will be needing this step; when we are about to use the zap config file. As it is stored in the repository.

Shall I remove the checkout for this PR

Comment (thc202) [reply .github/workflows/zap-scan.yml:10]: Right, better to leave it then.

**Pull request merged**